lib/ros/be/infra/cluster/generator.rb

```diff
@@ -30,7 +30,8 @@ def init_aws(cli)
                 STDOUT.puts "missing #{credentials_file}"
                 return
               end
-              cmd_string = "aws eks update-kubeconfig --name #{name}"
+              infra.config.cluster.aws_profile.nil? || ENV['AWS_PROFILE'] ? profile = "" : profile = "--profile #{infra.config.cluster.aws_profile}"
```

Comment:
`AWS_DEFAULT_PROFILE` is also very common to use, so we should check both.

Reply:
done